### DISCUSSION TITLE
1/6

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#40] [Announcements]
# 1/6

SICP Python 中文翻译

https://sicp.csfive.works/1/6

**Comment by shedding-ash** [comment]
这节好难

**Comment by siehun** [comment]
逐渐成为我看不懂的样子

**Comment by flisrkr** [comment]
私以为这章最大的困难其实是辨认一个函数的返回值到底是数据还是函数（）尤其是习惯了C++那种变量与函数泾渭分明的逻辑之后，适应python这个名称指代一切的思想会很费力🥵
